terralab/.terralab-cli-config

```diff
@@ -1,13 +1,12 @@
 # Teaspoons service API URL
-TEASPOONS_API_URL=https://teaspoons.dsde-dev.broadinstitute.org
+TEASPOONS_API_URL=https://teaspoons.dsde-prod.broadinstitute.org
 
 # Port to use for local server (for auth)
 SERVER_PORT=10444
 
 # Oauth config stuff
-OAUTH_OPENID_CONFIGURATION_URI=https://terradevb2c.b2clogin.com/terradevb2c.onmicrosoft.com/b2c_1a_signup_signin_dev/v2.0/.well-known/openid-configuration
-OAUTH_CLIENT_ID=bbd07d43-01cb-4b69-8fd0-5746d9a5c9fe
+OAUTH_OPENID_CONFIGURATION_URI=https://terraprodb2c.b2clogin.com/terraprodb2c.onmicrosoft.com/b2c_1a_signup_signin_prod/v2.0/.well-known/openid-configuration
+OAUTH_CLIENT_ID=bbd07d43-01cb-4b69-8fd0-5746d9a5c9fe # NEED TO UPDATE
```

Comment:
this needs to be updated once we have the new clientId (from terraform-ap-deployments PR)